qa/standalone/ceph-helpers.sh

```diff
@@ -170,7 +170,14 @@ function teardown() {
         fi
     fi
     if [ "$cores" = "yes" -o "$dumplogs" = "1" ]; then
-        display_logs $dir
+	if [ -n "$LOCALRUN" ]; then
+	    display_logs $dir
+        else
+	    # Move logs to where Teuthology will archive it
+	    # HACK: This path should come from teuthology code
```

Comment:
when run by teuthology as a workunit, several env vars are set - including TESTDIR, which is what you want as a prefix to archive/log here -- https://github.com/ceph/ceph/blob/master/qa/tasks/workunit.py#L452